code/modules/paperwork/paper.dm

```diff
@@ -29,64 +29,53 @@
 	var/list/offset_y[0] //usage by the photocopier
 	var/rigged = 0
 	var/spam_flag = 0
+	var/crumpled = FALSE
 
 	var/const/deffont = "Verdana"
 	var/const/signfont = "Times New Roman"
 	var/const/crayonfont = "Comic Sans MS"
 
-//lipstick wiping is in code/game/objects/items/weapons/cosmetics.dm!
-
-/obj/item/weapon/paper/New(loc, text, title)
-	..()
-	pixel_y = rand(-8, 8)
-	pixel_x = rand(-9, 9)
```

Comment:
Why remove this part? It helps keeping track of several paper sheets in one place.